### DISCUSSION TITLE
火狐浏览器：饼图svg模式，无法显示tooltip提示

### DISCUSSION THREAD

**Discussion by ixing7** [antvis/G2#5030] [Q&A]
- [ ] I have searched the [issues](https://github.com/antvis/g2/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://g2.antv.vision/zh/examples/case/pie#pie7](https://g2.antv.vision/zh/examples/case/pie#pie7)

### Steps to reproduce
1. 使用火狐浏览器打开DEMO链接：https://g2.antv.vision/zh/examples/case/pie#pie7
2. 在环形图页面，设置为svg渲染：renderer: 'svg'
3. 鼠标hover上去没有tooltip提示，应该显示tooltip提示窗。BUG复现完毕。

| Environment | Info |
|---|---|
| g2 | 4.1.5 |
| System | win10 |
| Browser | Firefox Browser 84.0.2 (64 位) |

---
IE、Chrome都能显示tooltip提示，唯独火狐出现了这个BUG，望修复，感谢❀❀❀

有的正常，有的显示不出来。

**Comment by hustcc** [comment]
因为底层 G 渲染引擎就版本对于 SVG 有不少 bug，并且架构上不是特别好修复，但是这些在 5.x 版本中会得到修复，这是一个[案例](https://g2.antv.antgroup.com/examples/renderer/renderer/#svg)。
